File: Algorithm/Programmers/Level1/solution(14).py
# 문자열 내 p와 y의 개수

def solution(s):
    answer = True
    count = 0
    s = s.lower()
    for i in s:
        if i == "p":
            count += 1
        elif i== "y":
            count -= 1
    if count != 0:
        answer = False

    return answer

File: Algorithm/Programmers/Level1/solution(14).java
// 문자열 내 p와 y의 개수

import java.util.*;

class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int ycount = 0;
        int pcount = 0;

        for (int i = 0; i < s.length(); i++){
            if (s.charAt(i) == 'y' || s.charAt(i) == 'Y')
                ycount++;
            else if (s.charAt(i) == 'p' || s.charAt(i) == 'P')
                pcount++;
        }
        
        if (ycount != pcount)
            answer = false;

        return answer;
    }
}